html/dashboard.html

```diff
@@ -3,124 +3,115 @@
-  padding: 10px 25px;
-  font-size: 24px;
-  text-align: center;
-  outline: none;
-  color: #fff;
-  background-color: #42c2f4;
-  border: none;
-  border-radius: 10px;
-  box-shadow: 0 4px #999;
-  position:relative;
+<button id="back">back</button>
+
+<div id="view"></div>
+<div id="view-rhs"></div>
+</div>
+
+<style>
+  #back{
+    padding: 10px 25px;
+    font-size: 24px;
+    text-align: center;
+    outline: none;
+    color: #fff;
+    background-color: #42c2f4;
+    border: none;
+    border-radius: 10px;
+    box-shadow: 0 4px #999;
+    position:relative;
   }
-
-    #back:hover {
-      background-color: #4192f4
-    }
-
-    #back:active {
-      background-color: #3e8e41;
- 
-    }
-    #view{
-      float:left;
-      padding-left:10px;
-    }
-    #view-rhs{
-      float:right;
-      padding-right: 10px;
+  #back:hover {
+    background-color: #4192f4
+  }
+  #back:active {
+    background-color: #3e8e41;
   }
- 
+  #view{
+    float:left;
+    padding-left:10px;
+  }
+  #view-rhs{
+    float:right;
+    padding-right: 10px;
+  } 
+</style>
 
-  </style>
 <script type="text/javascript">
-let my_spec = {
-  "$schema": "https://vega.github.io/schema/vega/v4.json",
-  "width": 600,
-  "height": 1000,
-  "padding": 80,
-  "autosize": "fit",
-
-  "data": [
-    {
-      "name": "tree",
-      "values":[]
-    },
-    {
-      "name": "sunburst",
-       "source": "tree",
-      "transform": [
-
-        {
-          "type": "stratify",
-          "key": "addr",
-          "parentKey": "parent"
-        },
-        {
-          "type": "partition",
-          "field": "elapsed_ns",
-          "sort": {"field": "value"},
-          "size": [{"signal": " 2 * PI"}, {"signal": "width / 2.5"}],
-          "as": ["a0", "r0", "a1", "r1", "depth", "chidlren"]
-        }
-      ]
-    },
-    
-    {
-     "name": "table",
-     "source": "tree",
-     "transform": [
-        {
-          "type": "filter",
-          "expr": "datum.parent == filterlist"
-        }
-     ]
-    }
-  ],
-   "signals": [
-        {
-          "name": "filterlist",
-          "value": "[0,0]",
-           "on": [{"events": "click", "update": "invert('yscale', y())"}]
-   
-        }
-      ],
-
-  "scales": [
-    {
-      "name": "color",
-      "type": "ordinal",
-      "range": {"scheme": "tableau20"}
-    },   
-    {
+  let my_spec = {
+    "$schema": "https://vega.github.io/schema/vega/v4.json",
+    "width": 600,
+    "height": 1000,
+    "padding": 80,
+    "autosize": "fit",
+    "data": [
+      {
+        "name": "tree",
+        "values":[]
+      },
+      {
+        "name": "sunburst",
+        "source": "tree",
+        "transform": [
+          {
+            "type": "stratify",
+            "key": "addr",
+            "parentKey": "parent"
+          },
+          {
+            "type": "partition",
+            "field": "elapsed_ns",
+            "sort": {"field": "value"},
+            "size": [{"signal": " 2 * PI"}, {"signal": "70*PI"}],
```

Comment:
Why this change? I.e. why is the radius of the circle a number multiplied by PI?
Should this be an absolute number in pixels?

Reply:
it's a typo while copying & pasting. Intended to replace any fixed number. I.e:  "size": [{"signal": "2 * PI"}, 700], or  "size": [{"signal": "2 * PI"}, {"signal": "width / 2"}],